LiteFarmOrg/LiteFarm
# Added validation of reading type in the add reading api

To Test,


1. Add a sensor to the farm.
2. call the add readings API of sensors.
3. make changes in the parameter_category to see parameter_category validation.

<img width="1071" alt="Screen Shot 2022-12-13 at 5 11 52 PM" src="https://user-images.githubusercontent.com/20675885/207480943-c02500ca-15ca-482d-8905-4c0160031c9c.png">

<img width="1077" alt="Screen Shot 2022-12-13 at 5 12 43 PM" src="https://user-images.githubusercontent.com/20675885/207481007-9329ffd0-1745-4549-9c36-3bb99a8b9f7c.png">
